ipfs-websites/pin-websites.sh

```diff
@@ -51,8 +51,8 @@ for s in $websites; do
             pinned=yes
         fi
         if [[ -n "$oldcid" && ("$newcid" != "$oldcid") ]]; then
-            echo "unpinning old version: $oldcid"
-            ipfs-cluster-ctl $@ pin rm --no-status "$oldcid"
+            echo "unpinning old version after 48h: $oldcid"
+            ipfs-cluster-ctl $@ pin add --expire-in '172800s' --no-status --name "${s}-outdated-$(date --utc -Iseconds)" "$oldcid"
```

Comment:
should be able to write `48h` rather than seconds.

Reply:
fixed